EnMasseProject/enmasse
# Go 1.12 update

A breakup of https://github.com/EnMasseProject/enmasse/pull/2948/ that only includes the fix for https://github.com/EnMasseProject/enmasse/issues/2486

# How to Test 
Ensure you have go 1.12+ installed and run `make`.  Without this PR the vet tool will fail to run, with the vet tool it will run.
